index.html

```diff
@@ -4998,6 +5000,18 @@ <h2 id="AudioWorkletGlobalScope">
             <code>registerProcessor</code> method is called, but this storage
             is internal and is not directly exposed to the user.
           </p>
+          <div class="note">
+            The <a>AudioWorkletGlobalScope</a> may also contain any other data
+            and code to be shared by these instances. As an example, multiple
+            processors might share an ArrayBuffer defining a wavetable or an
+            impulse response.
+          </div>
+          <div class="note">
+            Every <a>AudioWorkletGlobalScope</a> is associated with at most one
```

Comment:
How is this enforced (association to at most one AudioContext)? Can't seem to find a link for that, so if it exists a link to that would be good.